src/test/java/org/minions/devfund/damianvp/moviedatabase/MovieTest.java

```diff
@@ -0,0 +1,139 @@
+package org.minions.devfund.damianvp.moviedatabase;
+
+import java.util.ArrayList;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * class to test the Movie class.
+ */
+public class MovieTest {
+
+    private Actor actor1;
+    private Actor actor2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "actor2" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)